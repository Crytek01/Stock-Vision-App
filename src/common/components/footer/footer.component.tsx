import { FunctionComponent } from "react";
import styles from "./footer.module.css";

export const Footer: FunctionComponent = () => {
  return (
    <footer className={styles.footer}>
      <p>
        &copy; {new Date().getFullYear()} Stock Vision. All Rights Reserved.
      </p>

      <p className={styles.footerLastText}>Made with ❤️ by Erick Badilla</p>
    </footer>
  );
};
