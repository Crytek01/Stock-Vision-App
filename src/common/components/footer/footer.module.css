.footer {
  background-color: #333;
  color: #fff;
  text-align: center;
  padding: 1rem 0;
  width: 100%;
}

.footerLastText {
  margin-bottom: 0;
}
