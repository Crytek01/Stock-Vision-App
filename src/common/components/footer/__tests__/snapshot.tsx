import { render } from "@testing-library/react";
import { Footer } from "../footer.component";
it("renders footer component", () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
