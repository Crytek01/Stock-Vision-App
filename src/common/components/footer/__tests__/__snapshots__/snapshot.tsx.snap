// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer component 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      © 
      2023
       Stock Vision. All Rights Reserved.
    </p>
    <p
      class="footerLastText"
    >
      Made with ❤️ by Erick Badilla
    </p>
  </footer>
</div>
`;
